src/Apps/Order/Routes/Shipping2/index.tsx

```diff
@@ -68,15 +70,61 @@ const ShippingRouteLayout: FC<Omit<ShippingProps, "dialog">> = ({
   order,
 }) => {
   const shippingContext = useShippingContext()
+  const { handleSaveFulfillmentDetails } = useHandleSaveFulfillmentDetails()
 
-  const isOffer = order.mode === "OFFER"
+  const { jumpTo } = useJump()
+
+  useEffect(() => {
+    if (shippingContext.state.stage === "shipping_quotes") {
+      jumpTo("shippingOptionsTop", { behavior: "smooth" })
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [shippingContext.state.stage])
+
+  /*
+   * Re-save fulfillment details on load if they are already saved &
+   * require artsy shipping to advance to shipping quotes
+   */
+  useEffect(() => {
+    const { savedFulfillmentDetails } = shippingContext.orderData
+
+    const isArtsyShippingSaved =
```

Comment:
with index now a little cleaner these two effects feel much more at home at the top level.